README.md

```diff
@@ -1,128 +1,201 @@
-# sneaky-package 🥸  
-A ready-to-deploy Python package designed to **stealthily integrate files** within a machine, ensuring discreet and seamless file operations without detection. This project is nothing more than a *lightweight & faster* Python package that aims to highlight and improve security by demonstrating potential vulnerabilities in the system when connected with **`pip`**.
+# SNEAKY PACKAGE 👾
+![GSSoC-ext](GSSoC-Ext.png)
+## Table of Contents
+1. [🚀Project Overview](#project-overview)
+2. [✨ Key Features ✨](#✨key-features✨)
+3. [⛓️ What is a **Pip Package**? ⛓️](#⛓️what-is-a-pip-package-⛓️)
+4. [🪪 Licence](#🪪licence)
+5. [🚀 How to Contribute to This Project](#🚀-how-to-contribute-to-this-project)
+   - [🍴 Fork the Repository](#🍴-fork-the-repository)
```

Comment:
How to contribute section includes all these points, kindly refrain from adding unnecessary sections. This will make the readme cluttered. 
Instead, go for Broader sections and include info under them as subsection.